components/alert.css

```diff
@@ -0,0 +1,56 @@
+:root {
```

Comment:
You can placed all these css variable in main css file,  don't need to use another :root for that.

Reply:
Yep